Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.messages.in

```diff
@@ -22,7 +22,7 @@
 
 #if ENABLE(GPU_PROCESS)
 
-messages -> RemoteDisplayListRecorder NotRefCounted Stream {
+messages -> RemoteDisplayListRecorder NotRefCounted Stream AllowedWhenWaitingForSyncReply {
```

Comment:
I think in WebKit currently we can make Stream imply AllowedWhenWaitingForSyncReply

Reply:
ack. That's now the case - I must of missed this one. 

Currently Stream ==> AllowedWhenWaitingForSyncReply (for now) but AllowedWhenWaitingForSyncReply !==> Stream